catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/endpoint/OpenSearchEndpoint.java

```diff
@@ -138,6 +136,8 @@ public OpenSearchEndpoint(CatalogFramework framework, FilterBuilder filterBuilde
    * @param type Specifies the type of data to search for. (example: nitf)
    * @param versions Specifies the versions in a comma-delimited list.
    */
+  @SuppressWarnings("squid:S3776" /* high cognitive complexity but better readability */)
```

Comment:
What are your thoughts on refactoring this?

Reply:
Depending on how much over the limit the complexity is, you might be able to create simple private methods that are responsible for smaller portions. This would make it more readable and reduce the overall complexity.

Reply:
Removed